packages/framer-motion/src/animation/sequence/utils/sort.ts

```diff
@@ -0,0 +1,12 @@
+import { AbsoluteKeyframe } from "../types"
+
+export function compareByTime(
+    a: AbsoluteKeyframe,
+    b: AbsoluteKeyframe
+): number {
+    if (a.at === b.at) {
+        return a.value === null ? 0 : -1
```

Comment:
I thought this should be
```suggestion
        return a.value === null ? 1 : 0
```